docs/crashlytics/overview.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Crashlytics
+sidebar_label: Overview
+---
+
+##  What does it do?
+
+Crashlytics helps you to collect analytics and details about crashes and errors that occur in your app. It does this through three aspects:
+
+- **Logs**: Log events in your app to be sent with the crash report for context if your app crashes.
+- **Crash reports**: Every crash is automatically turned into a crash report and sent when the application next opens.
+- **Stack traces**: Even when an error is caught and your app recovers, the Dart stack trace can still be sent.
+
+<YouTube id="k_mdNRZzd30"/>
+
+## Installation
+
+### 1. Add dependency
+
+```yaml {5} title="pubspec.yaml"
+dependencies:
+  flutter:
+    sdk: flutter
+  firebase_core: "^{{ plugins.firebase_core }}"
```

Comment:
Haven't been following the updates to `firebase_core` so I may have missed something but is this step needed as won't it get pulled in as a transitive dependency?